extensions/terminal-suggest/src/terminalSuggestMain.ts

```diff
@@ -232,7 +229,7 @@ export async function getCompletionItemsFromSpecs(
 	let hasCurrentArg = false;
 	let fileExtensions: string[] | undefined;
 
-	let precedingText = terminalContext.commandLine.slice(0, terminalContext.cursorPosition + 1);
+	let precedingText = terminalContext.commandLine.slice(0, terminalContext.cursorPosition + 1).replaceAll('\n', '');
```

Comment:
The prompt input model is meant to report new lines correctly, ie. Removing all new lines from the buffer, but keeping them in the presence of a line continuation (`...`, `>`, etc.)

Reply:
oh, it isn't doing that currently.